api/accounts/models/user.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from api.models import BaseModel
+
+
+class User(AbstractUser, BaseModel):
```

Comment:
The `AbstractUser` model provides certain built in fields, such as `password`, `last_logged_in` etc. This means we don't need to handle the hashing of the password ourselves